exercises/react/example.cr

```diff
@@ -0,0 +1,71 @@
+alias CellValue = Int32
+
+private class Cell
+  @dependents = [] of ComputeCell
+
+  def initialize(initial : CellValue)
+    @value = initial
+  end
+
+  getter :value
+
+  protected getter :dependents
+  protected def propagate; _propagate; end
```

Comment:
whaaaat? Now that I removed `Reactor` I suddenly had to start doing this protected that delegates to a private for some reason. If I don't do this and just have `protected def propagate` in `ComputeCell`, I get `protected method 'propagate' called for ComputeCell` despite the fact that I didn't change anything else. Why do I have to do this now that I removed `Reactor`? What changed?